themes/default/content/blog/tour-pulumi-equinix-provider/index.md

```diff
@@ -0,0 +1,165 @@
+meta_desc: Learn to use the capabilities of the Pulumi Equinix Provider, including how to deploy Kubernetes on Equinix Metal.
+meta_image: tour-equinix-provider.png
+authors:
+    - josh-kodroff
+    - oscar-cobles
+tags:
+    - equinix
+    - kubernetes
+---
+
+Equinix recently released their self-maintained, fully-supported Pulumi provider, available in the [Pulumi Registry](https://www.pulumi.com/registry/packages/equinix/). In this post, you'll get an overview of the Equinix resources the provider can manage and we'll show you how to deploy a Kubernetes cluster and associated workloads on Equinix Metal.
+
+<!--more-->
+
+## Introducing the Equinix Provider
+
+The Equinix provider can manage resources for:
+
+- [Equinix Metal](https://www.equinix.com/products/digital-infrastructure-services/equinix-metal), which provides high-performance, bare-metal compute resources.
+- [Equinix Fabric](https://www.equinix.com/products/digital-infrastructure-services/equinix-fabric), which provides software-defined networking that enables low-latency connections between private networks; select service providers like Salesforce, AWS, Azure, and more; and Equinix Metal devices.
+- [Equinix Network Edge](https://www.equinix.com/products/digital-infrastructure-services/network-edge), which enables organizations to deploy virtual network functions (VNFs) (like virtual firewall devices) in a centralized point, greatly simplifying network traffic management for multi- and hybrid cloud scenarios.
+
+Detailed documentation for the Equinix provider can be found in the [Pulumi Registry](https://www.pulumi.com/registry/) along with 130+ (at the time of writing) other providers that can be used to manage cloud and SaaS resources.
+
+{{% notes type="info" %}}
+**Note:** The Equinix provider effectively replaces the now-deprecated [Equinix Metal provider](https://www.pulumi.com/registry/packages/equinix-metal/). To migrate your Pulumi resources to the new provider, use the [`pulumi import`](https://www.pulumi.com/docs/cli/commands/pulumi_import/) command.
+{{% /notes %}}
+
+## Creating a Kubernetes cluster on Equinix Metal
+
+In order to demonstrate the power and utility of Pulumi and the Equinix provider, Equinix Labs has produced a codebase that [creates a Kubernetes cluster on Equinix Metal](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/). The codebase is available in both [TypeScript](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/tree/main/nodejs) and [Python](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/tree/main/python).
+
+{{% notes type="info" %}}
+For an overview of how Pulumi works along with a guided tour of the codebase and deploying a workload onto the Kubernetes cluster, check out [Pulumi's presentation at Equinix Demo Day 2023](https://youtu.be/-siv1ga0l_o). (Pulumi's presentation begins at 3:30:00 below, or click the preceding link to jump directly to Pulumi's presentation on YouTube.):
```

Comment:
We can use this link to the exact starting point: https://www.youtube.com/live/-siv1ga0l_o?feature=share&t=12600